test/engine/RuleEngineSpec.scala

```diff
@@ -87,15 +86,15 @@ class RuleEngineSpec extends UnitSpec with MockitoSugar with WithFakeApplication
       //when
       val maybeLocation: Future[Option[Location]] = new RuleEngine {
         override val rules: List[Rule] = List(firstRule, secondRule)
-      }.getLocation(mock[AuthContext], mock[RuleContext], auditContext)(request, hc)
+      }.getLocation(mock[RuleContext], auditContext)(request, hc)
 
       //then
       val location: Option[Location] = await(maybeLocation)
       location shouldBe Some(expectedLocation)
 
       //then
-      verify(firstRule).apply(any[AuthContext], any[RuleContext], any[AuditContext])(eqTo(request), eqTo(hc))
-      verify(secondRule, never()).apply(any[AuthContext], any[RuleContext], any[AuditContext])(any[Request[AnyContent]], any[HeaderCarrier])
+      verify(firstRule).apply( any[RuleContext], any[AuditContext])(eqTo(request), eqTo(hc))
+      verify(secondRule, never()).apply(any[RuleContext], any[AuditContext])(any[Request[AnyContent]], any[HeaderCarrier])
```

Comment:
instead of `verify(secondRule, never())` you could do `verifyZeroInteractions` or `verifyNoMoreInteractions`